torchao/quantization/quant_primitives.py

```diff
@@ -766,13 +766,16 @@ def _choose_qparams_affine(
         assert mapping_type == MappingType.ASYMMETRIC.name
         scale = (max_val_pos - min_val_neg) / float(quant_max - quant_min)
         scale = torch.clamp(scale, min=eps)
-        if preserve_zero:
-            zero_point = quant_min - torch.round(min_val_neg / scale)
-            zero_point = torch.clamp(zero_point, quant_min, quant_max)
+        if zero_point_domain is None:
```

Comment:
@jerryzh168 confirm if this is OK.  It was needed to get scale-only quantization in affine_quantized_tensor

Reply:
OK, should zero_point be `None` here?

Reply:
I could make it None, but that changes the return type of this method from Tuple[Tensor, Tensor] to Tuple[Tensor, Optional[Tensor]]

Reply:
yeah I think making it None probably makes more sense here